pontoon/checks/apps.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class ChecksConfig(AppConfig):
```

Comment:
i think you need a line in __init__ to trigger this - see here for docs https://docs.djangoproject.com/en/2.0/ref/applications/#for-application-authors

on that note - it seems like some of the other apps have an appconfig that wont afaict be used